students/rgpag/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1,101):
+	if i%3 ==0 and i%5==0: 
+		print('FizzBuzz')
+	elif i%3==0:
+		print('Fizz')
+	elif i%5==0:
+		print('Buzz')
+	else:
+		print(i)
```

Comment:
It's very important in Python that you configure your editor to use 4 space characters when you hit the tab key. A Pep8 linter will catch this as well, and there are instructions in the course materials for configuring this setting in Sublime Text.